ui/components/PollingIndicator.tsx

```diff
@@ -0,0 +1,60 @@
+import { CircularProgress, Tooltip } from "@material-ui/core";
+import * as React from "react";
+import styled from "styled-components";
+import { theme } from "..";
+import { muiTheme } from "../lib/theme";
+import Flex from "./Flex";
+import Spacer from "./Spacer";
+
+type Props = {
+  /** whether `<CircularProgress />` should be shown */
+  loading: boolean;
+  interval: string;
+  className?: string;
+};
+
+function PollingIndicator({ className, loading, interval }: Props) {
+  const date = new Date();
+  const [updated, setUpdated] = React.useState(date.toTimeString().slice(0, 8));
+  const [indicator, setIndicator] = React.useState(false);
```

Comment:
I would use visible, setVisible here since we're in the PollingIndicator component so we know what we're referencing by that attribute